static/templyte/src/app/download-template/download-template.component.html

```diff
@@ -0,0 +1,39 @@
+<mat-toolbar class="mat-elevation-z3">
+    <nav class="navbar">
+        <ul class="nav navbar-right">
+            <li>
+                <button mat-button (click)="goHome()">Home</button>
+            </li>
+            <li>
+                <button mat-button (click)="createTemplate()">Create Template</button>
+            </li>
+            <li>
+                <button mat-button (click)="myTemplates()">My Templates</button>
+            </li>
+            <li>
+                <button mat-button (click)="logout()">Logout</button>
+            </li>
+        </ul>
+    </nav>
+</mat-toolbar>
+
+<mat-card class="card-download">
+    <h1>Download a Template</h1>
+
+    <h1>{{(templateDirectoryInfo | async)?.templateName}}</h1>
+    <!--TODO: Add description-->
```

Comment:
Who was meant to add the description? It shouldn't take too long so we should get it done ASAP, preferably in a separate PR.